html/controls/mobile.html

```diff
@@ -125,20 +122,41 @@ <h1>Jam Control</h1>
 		<button class="Timeout" data-role="button">Timeout</button>
 		<div class="ui-grid-b Timeout">
 			<div class="ui-block-a"><button class="Team1">
+				TO 
 				<span class="Team1 Name">Team 1</span>
 				<span class="Team1 AlternateName"></span>
+				
 			</button></div>
-			<div class="ui-block-b"><button class="Official">Official</button></div>
+			<div class="ui-block-b"><button class="Official">TO Official</button></div>
```

Comment:
I've already heard this as "official time out" rather than "time out official". Is this one of those things that varies across leagues?

Reply:
Official Timeout is definitely the preferred nomenclature, but using "TO Official" was more about keeping the buttons consistent with "TO Team 1" and "TO Team 2".  Also when the buttons are small, the text is truncated, so I wanted to make it clearer it was a TO instead of an OR (so it shows up as "TO Off..." instead of "Offici..." or the like)